parsers/verilog.c

```diff
@@ -810,6 +841,10 @@ static void createTag (tokenInfo *const token, verilogKind kind)
 		verbose ("Class %s extends %s\n", vStringValue (token->name), tag.extensionFields.inheritance);
 	}
 
+	if (token->parameter)
+		attachParserField (&tag, false,
+						   VerilogFields [F_OVERRIDABLE_PARAMETER].ftype, "");
+
```

Comment:
(Attaching "" as a value is o.k. even if we define the filed type as a boolean.)